pom.xml

```diff
@@ -387,6 +387,11 @@
           <artifactId>hadoop-mapreduce-client-common</artifactId>
           <version>2.7.0-mapr-1602</version>
         </dependency>
+        <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-hdfs-client</artifactId>
+          <version>2.7.0-mapr-1602</version>
```

Comment:
What is this dependency for?

Reply:
I have added this dependency to make the mapr pom dependencies to be in sync with the mlcp dependencies. I have also tested this addition with the 3.3.0 version in the mapr, it was not problematic as there is no code activation for mapr.